<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RenderEditor</class>
 <widget class="QDialog" name="RenderEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>528</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QPushButton" name="pushButtonApply">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>&amp;Apply</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonReset">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButtonClose">
     <property name="text">
      <string>&amp;Close</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="General">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Solid Color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushButtonSolidColor">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Edge Color</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushButtonEdgeColor">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_15">
         <property name="text">
          <string>Background Color</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="pushButtonBackgroundColor">
         <property name="text">
          <string>Set</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCheckBox" name="checkBoxUseLog">
         <property name="text">
          <string>Use Logarithmic Scale</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ColorBar">
      <attribute name="title">
       <string>Color Bar</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>General</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Aspect Ratio</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="doubleSpinBoxAspectRatio"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>Labels</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxLabels"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxTitle">
         <property name="title">
          <string>Title Font</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Text</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEditTitle"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Font</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxTitleFonts"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxTitleSize"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonTitleColor">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxTitleShadow">
            <property name="text">
             <string>Use Shadow</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxLabels">
         <property name="title">
          <string>Label Font</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Font</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="comboBoxLabelsFonts"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Size</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBoxLabelSize"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonLabelsColor">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBoxLabelsShadow">
            <property name="text">
             <string>Use Shadow</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Axes">
      <attribute name="title">
       <string>Axes</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBoxCoordinates">
         <property name="title">
          <string>Coordinates</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item>
             <widget class="QCheckBox" name="checkBoxShowXAxis">
              <property name="text">
               <string>Show X Axis</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxShowYAxis">
              <property name="text">
               <string>Show Y Axis</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxShowZAxis">
              <property name="text">
               <string>Show Z Axis</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAxesLines">
              <property name="text">
               <string>Show Lines</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAxesInnerLines">
              <property name="text">
               <string>Show Inner Lines</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAxesLabels">
              <property name="text">
               <string>Show Axes Labels</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAxesTicks">
              <property name="text">
               <string>Show Axes Ticks</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBoxAxesMinorTicks">
              <property name="text">
               <string>Show Axes Minor Ticks</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="1" column="0">
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Label Format</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEditLabelFormat"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Fly Mode</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="comboBoxFlyMode"/>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_13">
              <property name="text">
               <string>Corner Offset</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QDoubleSpinBox" name="doubleSpinBoxCornerOffset"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>Unit</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEditUnit"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Color</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QPushButton" name="pushButtonColorAxes">
              <property name="text">
               <string>Set</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Segments">
      <attribute name="title">
       <string>Segments</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Set Segment Specific Visibility</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidgetSegments"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>checkBoxUseLog</tabstop>
  <tabstop>doubleSpinBoxAspectRatio</tabstop>
  <tabstop>spinBoxLabels</tabstop>
  <tabstop>lineEditTitle</tabstop>
  <tabstop>comboBoxTitleFonts</tabstop>
  <tabstop>spinBoxTitleSize</tabstop>
  <tabstop>pushButtonTitleColor</tabstop>
  <tabstop>checkBoxTitleShadow</tabstop>
  <tabstop>comboBoxLabelsFonts</tabstop>
  <tabstop>spinBoxLabelSize</tabstop>
  <tabstop>pushButtonLabelsColor</tabstop>
  <tabstop>checkBoxLabelsShadow</tabstop>
  <tabstop>pushButtonApply</tabstop>
  <tabstop>pushButtonReset</tabstop>
  <tabstop>pushButtonClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
